src/ui/core/zcl_abapgit_html.clas.abap

```diff
@@ -29,6 +29,11 @@ CLASS zcl_abapgit_html DEFINITION
         iv_checked     TYPE abap_bool OPTIONAL
       RETURNING
         VALUE(rv_html) TYPE string .
+    CLASS-METHODS data_attr
```

Comment:
suggest renaming this method to `parse_data_attr `

Reply:
done